oxidation/libparsec/crates/platform_async/Cargo.toml

```diff
@@ -10,6 +10,11 @@ futures = "0.3.21"
 
 [dev-dependencies]
 log = "0.4.17"
+# `std::time::Instant` is not available for wasm32, crate `instant` provides a
+# fallback based on `window.performance.now()`
+instant = "0.1.12"
```

Comment:
Shouldn't we re-export this `crate` to expose it for crates using `platform-async` ?

Reply:
Yes, I think that's something we want to do in the long run, but I'd rather wait a bit for that until we have an actual use (or at least until we have a better view on what features we need to export, this is to avoid a "export everything every-time you need something" reflex that may produce too much coupling between crates)

Reply:
Ok